### PR TITLE
Added "inc" command for incrementing a key.

### PR DESCRIPTION
**Pull request by petermattis**
Enhanced "get" and "scan" to properly display integer values.

### CODE REVIEW

**Comment by bdarnell** [comment]
LGTM

File: server/cli/kv.go [hunk 1]
```diff
@@ -124,6 +129,50 @@ func runPut(cmd *commander.Command, args []string) {
...
+	Flag: *flag.CommandLine,
+}
+
+func runInc(cmd *commander.Command, args []string) {
+	if len(args) > 2 {
+		cmd.Usage()
+		return
+	}
+
+	if strings.HasPrefix(args[0], "\x00") {
```
Comment (spencerkimball) [inline server/cli/kv.go]: This seems like the wrong place for such a check...

File: server/cli/kv.go [hunk 2]
```diff
@@ -170,7 +219,7 @@ func runDel(cmd *commander.Command, args []string) {
 // range.
 var CmdScan = &commander.Command{
 	UsageLine: "scan [options] [<start-key> [<end-key>]]",
-	Short:     "scans a range of keys",
+	Short:     "scans a range of keys\n",
```
Comment (spencerkimball) [inline server/cli/kv.go]: Why the \n suffix? We don't have that elsewhere.

File: server/cli/range.go [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+		SplitKey: splitKey,
+	}
+	resp := &proto.AdminSplitResponse{}
+	if err := kv.Call(proto.AdminSplit, req, resp); err != nil {
+		fmt.Fprintf(os.Stderr, "split failed: %s\n", err)
+		os.Exit(1)
+	}
+}
+
+// A CmdSplitMerge command merges a range.
```
Comment (spencerkimball) [inline server/cli/range.go]: s/CmdSplitMerge/CmdMergeRange/

File: server/cli/kv.go (hunk 2 above)
Comment (petermattis) [inline server/cli/kv.go]: It's a hack to make `cockroach help` look better by providing separators between groups of related commands.

File: server/cli/range.go (hunk 3 above)
Comment (petermattis) [inline server/cli/range.go]: Oops. I'll fix.

File: server/cli/kv.go (hunk 1 above)
Comment (petermattis) [inline server/cli/kv.go]: Problematic how? Mucking with system keys accidentally seems bad.

**Pull request merged**